lib/src/main/java/io/ably/lib/realtime/AblyRealtime.java

```diff
@@ -50,6 +51,7 @@ public AblyRealtime(ClientOptions options) throws AblyException {
 		super(options);
 		channels = new Channels();
 		connection = new Connection(this);
+		http.setHost(options.realtimeHost);
```

Comment:
I see two problems with this approach:

1. You are setting the REST Host to the RealtimeHost in all situations. This is wrong, we only use the fallback host for REST requests when the realtime host is using that fallback host.
2. If a realtime request falls back to a fallback host, but then later resumes using the default realtime host, all future REST requests will continue to go to the incorrect fallback host, see https://github.com/ably/ably-java/pull/80/files#diff-060685a947a7032665de7ac9d33bb6ffR588